jekyll/_cci2/server-3-operator-nomad.adoc

```diff
@@ -0,0 +1,118 @@
+---
+version:
+- Server v3.x
+- Server Admin
+---
+= Introduction to Nomad Cluster Operation
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn how to operate the Nomad Cluster in your CircleCI 2.0 installation.
+:icons: font
+:toc: macro
+:toc-title:
+
+CircleCI uses https://www.hashicorp.com/blog/nomad-announcement/[Nomad] as the primary job scheduler. This chapter provides a basic introduction to Nomad for understanding how to operate the Nomad Cluster in your CircleCI installation.
```

Comment:
A tiny point, but the capitalization of Nomad Cluster/cluster and Job/job is inconsistent.